recipe/build.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
 
+source activate "${CONDA_DEFAULT_ENV}"
+
 chmod +x configure
 
-./configure --prefix=$PREFIX --enable-cxx
+if [ "$(uname)" == "Darwin" ];
+then
+    ./configure --prefix=$PREFIX --enable-cxx --build=x86_64-apple-darwin
```

Comment:
Doesn't work in travis. `--enable-fat` is not supported on all architectures, but if it builds on a particular platform it should work on all x86 architectures.